### DISCUSSION TITLE
skip keyboard aura initialisation on startup

### DISCUSSION THREAD

**Discussion by elfi-ox** [seerge/g-helper#2898] [Q&A]
Hello, i've read a lot of question here to see if people with the same problems as me had found an answer and one I saw a lot was about conflict between Aura and G-helper RGB settings. So everytime someone ask about this you answered with https://github.com/seerge/g-helper#skip-keyboard-aura-initialisation-on-startup. I may be missing something or maybe it's just outdated but this link bring me into a webpage that is literrally just https://github.com/seerge/g-helper and I don't see where I can disable it. It usually dosen't bother me that much since my aura creator settings take over during boot so i have a nice rainbow effect followed by my actuals settings, however i've noticed since installing g-helper that my lightbar is often overwrited with an almost static blue/purple until I reaply my aura creator settings. Here is an exemple of how it looks like https://imgur.com/a/rCPNDE7
I think disabling this "aura-initialisation-on-startup" might help

**Comment by seerge** [comment]
@elfi-ox hello, this is now part of Power User settings section 
https://github.com/seerge/g-helper/wiki/Power-user-settings#skip-keyboard-aura-initialisation-on-startup

**Reply by elfi-ox** [reply]
Thanks you this is very helpful
